index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/reset.css">
+    <link rel="stylesheet" href="css/utility.css">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Micro-Portfolio</title>
+</head>
+<body>
+    <header class="container bg-gray">
+        <div class="flex header">
+            <a href="#" class="text-light">John Doe</a>
+            <button class="btn-icon" type="button"> <img src="assets/img/menu.png" alt="menu"></button>
```

Comment:
- [ ] Please nest the #hamburger menu# button withing `<nav>` tag. This allows screen reader users to navigate directly to the navigation landmark. Kindly follow the requirement from the previous review.